src/operations/find.ts

```diff
@@ -77,13 +77,10 @@ export class FindOperation extends CommandOperation<Document> {
   override options: FindOptions & { writeConcern?: never };
   filter: Document;
 
-  constructor(
-    collection: Collection | undefined,
-    ns: MongoDBNamespace,
-    filter: Document = {},
-    options: FindOptions = {}
-  ) {
-    super(collection, options);
+  public encryptionEnabled = false;
```

Comment:
why is this necessary?  without this, when encryption is enabled:

- the find operation will provide a MonogoDBResponse to the server
- the server will provide it to the connection
- the crypto connection won't pass it along, so a raw response is returned

Reply:
You're right, I think I did this in parallel with removing the pass through for crypto and didn't go back, good catch ty!